README.md

```diff
@@ -4,12 +4,12 @@
 <img alt="Build Status" src="https://prow.k8s.io/badge.svg?jobs=ci-kind-build">
 </a> <a href="https://godoc.org/sigs.k8s.io/kind"><img src="https://godoc.org/sigs.k8s.io/kind?status.svg"></a> <a href="https://goreportcard.com/report/sigs.k8s.io/kind"><img alt="Go Report Card" src="https://goreportcard.com/badge/sigs.k8s.io/kind" /></a></p>
 
-### [View The Documentation](https://kind.sigs.k8s.io)
+# Please see [Our Documentation](https://kind.sigs.k8s.io/docs/user/quick-start/) for more in-depth installation etc.
```

Comment:
- users looking for docs from the should really start at this page, the root page of the site is a landing page nearly 1:1 with the readme
- this should be bigger, ideally many potential users will click this, however we should still have a readme because people don't like having to go to another site :-)